### PR TITLE
Add ListClusterServiceVersionInAllNamespaces method

### PR DESCRIPTION
**Pull request by mcornea**

### CODE REVIEW

File: pkg/olm/clusterserviceversionlist.go [hunk 1]
```diff
@@ -99,3 +99,47 @@ func ListClusterServiceVersionWithNamePattern(
...
+		passedOptions = options[0]
+		logMessage += fmt.Sprintf(" with the options %v", passedOptions)
+	}
+
+	glog.V(100).Infof(logMessage)
+
+	csvList, err := apiClient.ClusterServiceVersions("").List(context.Background(), passedOptions)
+
+	if err != nil {
+		glog.V(100).Infof("Failed to list CSVs in all namespace due to %s", err.Error())
```
Comment (yprokule) [inline pkg/olm/clusterserviceversionlist.go:126]: nit: `namespaces`
```suggestion
		glog.V(100).Infof("Failed to list CSVs in all namespaces due to %s", err.Error())
```

Comment (mcornea) [reply pkg/olm/clusterserviceversionlist.go]: Thanks, updated.

**Review by kononovn — APPROVED** [review]
lgtm

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**